src/StakeRegistry.sol

```diff
@@ -224,32 +246,37 @@ contract StakeRegistry is StakeRegistryStorage {
     }
 
     function setMinimumStakeForQuorum(
-        uint8 quorumNumber, 
+        uint8 quorumNumber,
         uint96 minimumStake
     ) public virtual onlyCoordinatorOwner quorumExists(quorumNumber) {
         _setMinimumStakeForQuorum(quorumNumber, minimumStake);
     }
 
     /**
-     * @notice Sets the stake type for the registry
+     * @notice Sets the stake type for the registry for a specific quorum
+     * @param quorumNumber The quorum number to set the stake type for
      * @param _stakeType The type of stake to track (TOTAL_DELEGATED, TOTAL_SLASHABLE, or BOTH)
      */
-    function setStakeType(StakeType _stakeType) external onlyCoordinatorOwner {
-        _setStakeType(_stakeType);
+    function setStakeType(uint8 quorumNumber, StakeType _stakeType) external onlyCoordinatorOwner {
```

Comment:
What happens if a user wants to transition a quorum from being TDD to TSS? Is this possible (seems like no given current RegCord implementation)? Is this a valid use case?

Reply:
Not currently supported but should be possible to switch between them if you update stakes for the quorum for everyone afterward